dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -324,18 +327,20 @@ public boolean equals(Contentlet contentlet)throws DotRuntimeException {
 		}
     }
 
-    @Override
-    public boolean equals(Object o) {
+	@Override
+	public boolean equals(Object o) {
+		if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00122)